src/system/Form/ToggleGroup.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { ThemeUIStyleObject } from 'theme-ui';
+
+/**
+ * Internal dependencies
+ */
+import { Box } from '../Box';
+import { Button } from '../Button';
+
+export interface ToggleGroupProps {
+	value?: string;
+	onChange?: ( value: string ) => void;
+	options?: {
+		value: string;
+		label: string;
+	}[];
+	sx?: ThemeUIStyleObject;
+}
+
+const ToggleGroup = ( props: ToggleGroupProps ) => {
+	const { options, value, onChange } = props;
+
+	const handleClick = ( e: React.MouseEvent< HTMLButtonElement >, val: string ) => {
+		e.preventDefault();
+		onChange?.( val );
+	};
+
+	return (
+		<Box
+			role="group"
+			dir="ltr"
+			tabIndex={ 0 }
```

Comment:
We should never use 0 or positive values; this will affect the page tabbing order. https://www.a11yproject.com/posts/how-to-use-the-tabindex-attribute/.

Also, you are enforcing an `ltr,` which we don't usually do. Removing this line is enough.